synapse/http/client.py

```diff
@@ -819,6 +820,127 @@ def __init__(
             )
 
 
+class SimpleReplicationClient(BaseHttpClient):
+    """No frills client for connecting to Replication endpoints.
+
+    Uses existing BaseHttpClient methods but replaces the 'agent' used to make the
+    request with one that supports HTTP and HTTPS.
```

Comment:
Perhaps:
`Uses existing BaseHttpClient methods and creates a custom 'agent' used to make the request with one that supports HTTP and HTTPS` ?

Understand, that in the next iteration of this `Client`, it will have Unix socket support added, which will touch this string again.

Reply:
See my comment there. 😄